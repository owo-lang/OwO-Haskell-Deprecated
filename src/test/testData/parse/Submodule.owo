module Parsing.Submodule where
  insideTop : Type
  module modA where
    insideA : Type
  insideTop : Type
  module modB where
    insideB : Type
    module modC where
      insideC : Type
    insideB : Type
  insideTop : Type
