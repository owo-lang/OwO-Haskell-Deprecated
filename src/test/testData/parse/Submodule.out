File type: CodeFileType
Top module name: .Parsing.Submodule
 type signature "insideTop" (33 2 3) (42 2 12)
  no pragmas
  named reference "Type" (45 2 15) (49 2 19)
 submodule .modA
  type signature "insideA" (74 4 5) (81 4 12)
   no pragmas
   named reference "Type" (84 4 15) (88 4 19)
 type signature "insideTop" (91 5 3) (100 5 12)
  no pragmas
  named reference "Type" (103 5 15) (107 5 19)
 submodule .modB
  type signature "insideB" (132 7 5) (139 7 12)
   no pragmas
   named reference "Type" (142 7 15) (146 7 19)
  submodule .modC
   type signature "insideC" (175 9 7) (182 9 14)
    no pragmas
    named reference "Type" (185 9 17) (189 9 21)
  type signature "insideB" (194 10 5) (201 10 12)
   no pragmas
   named reference "Type" (204 10 15) (208 10 19)
 type signature "insideTop" (211 11 3) (220 11 12)
  no pragmas
  named reference "Type" (223 11 15) (227 11 19)
